src/reducers/current.js

```diff
@@ -92,17 +92,17 @@ export const sGetCurrentFromUi = (state) => {
 
     switch (ui.type) {
         case VIS_TYPE_PIE:
-            return getPieCurrentFromUi(state, ui)
+            return getPieCurrentFromUi(state.current, ui)
```

Comment:
I found out we were passing the whole Redux state here instead of just the `current` slice like in other parts of the codebase.
This required to adjust the layout validation for Scatter where `ui` is needed and was read from this "current" object.

Reply:
ok.